cypress/integration/footer.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="Cypress" />
+
+describe('User can see a footer bar', () => {
+
+	it('User signs in successfully', () => {
```

Comment:
this it block should be named to whats going on, since we already have a "it" block names User signs in successfully

Reply:
Thanks!  Fixed.